app/src/main/res/values/dimens.xml

```diff
@@ -3,10 +3,6 @@
   <!-- Default screen margins, per the Android Design guidelines. -->
   <dimen name="activity_horizontal_margin">16dp</dimen>
   <dimen name="activity_vertical_margin">16dp</dimen>
-
-  <integer name="fade_in_duration_ms">1000</integer>
-  <integer name="fade_out_duration_ms">1000</integer>
```

Comment:
`fade_in` , `fade_out` you have shifted but you have not updated the implementation code where they are getting used and maybe there are not even getting used. So to be on save side, I suggest shifting it back to its original so that it does not create any issues for others.

Reply:
Ok, I'll update it